src/lib/colors/strategies/argb-strategy.ts

```diff
@@ -11,7 +11,7 @@ class ARGBColorExtractor implements IColorStrategy {
   public name: string = 'ARGB';
 
   private extractRGBValue(value): number[] {
-    let rgb: any = /#(.+)/gi.exec(value);
+    let rgb: any = /#?[0x]?[0X]?(.+)/gi.exec(value);
```

Comment:
It works but the regexp could be improved.
For instance the regexp is case insensitive, so we can remove `0X`.